configs/azure_c_shared_utilityFunctions.cmake

```diff
@@ -704,6 +704,8 @@ function(set_platform_files c_shared_dir)
         set(TICKCOUTER_C_FILE ${c_shared_dir}/adapters/tickcounter_linux.c PARENT_SCOPE)
         if (${use_default_uuid})
             set(UNIQUEID_C_FILE ${c_shared_dir}/adapters/uniqueid_stub.c PARENT_SCOPE)
+        elseif(AZSPHERE)
+            set(UNIQUEID_C_FILE ${c_shared_dir}/adapters/uniqueid_azsphere.c PARENT_SCOPE)
         else()
             set(UNIQUEID_C_FILE ${c_shared_dir}/adapters/uniqueid_linux.c PARENT_SCOPE)
         endif()
```

Comment:
I see AZSPHERE is not included @Line 719.  While not currently calling getenv, curious if Azure Sphere possibly would use it in the future?  

```
if (WIN32 OR MACOSX OR LINUX OR AZSPHERE)
```

Reply:
AZSPHERE will resolve under LINUX path in this settings